vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -666,3 +666,34 @@ user_attribute_name: sAMAccountName
 
 microservice_protocol: http
 
+# foia_extension_formatted is only used by the foia play
+foia_extension_formatted: "{{ arkcase_extension_install | ternary('-' ~ arkcase_extension_id, '') }}"
```

Comment:
FOIA isn't part of ArkCase CE (adding this comment since this file is "arkcase-ce-facts.yml").

Reply:
Exactly. Need to remove that.